### DISCUSSION TITLE
I am wondering if there is any way to [Apr 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5489] [Discord Threads]
> 🙋‍♂️ **macattach**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Using our API you can get the data in same format(Including all virtual columns) and in order to get at SQL level you have to construct the SQL query.  
#  
> 🙋‍♂️ **macattach**  
#### Thanks. I am able to construct a SQL query, however, when I try to select the formula column, I receive a "Column Does Not Exist" error message. Any guidance on that piece would be grearly appreciated. I will poke around some more with the API to try and extract it that way.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1083234206504726568)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
